Makefile

```diff
@@ -35,10 +34,6 @@ TAGS      :=
 LDFLAGS   := "-w -s -X 'k8s.io/cloud-provider-openstack/pkg/version.Version=${VERSION}'"
 REGISTRY ?= k8scloudprovider
 
-ifneq ("$(DEST)", "$(PWD)")
```

Comment:
we still need to run make from the base dir of project , why is this removed?

Reply:
I doesn't matter anymore. `go` automatically detects the `go.mod` inside the current directory and builds packages independently on the path.

Reply:
But for make its required , the below error is for make right, for make to run we need to be in base dir

Reply:
`make` command is usually executed within the current directory. This check is not needed, besides it [confuses end users](https://github.com/kubernetes/cloud-provider-openstack/issues/741#issuecomment-537359670), when they want to build the binary from source